src/workstation/cli/crud.py

```diff
@@ -180,6 +180,15 @@ def common_options(func):  # noqa: D103
     type=str,
     metavar="<str>",
 )
+@click.option(
+    "--envs",
+    "-e",
+    "envs",
```

Comment:
I'd have maybe expected it to be 'env' if it's passed multiple times each with one env var?

Reply:
That's fair, have debated if it should be plural or not but I think that makes more sense.

Reply:
Agree, I would have expected something like 

`--env VAR1=foo --env VAR2=bar`

Reply:
Have changed it, `=` not possible with how it is setup with click so it is space